### PR TITLE
Add profile for 4 days  MV backpressure longevity test

### PR DESCRIPTION
**Pull request by juliayakovlev**

### CODE REVIEW

File: tests/longevity-2mv-backpressure-4d.yaml [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+        gce_root_disk_type_monitor: 'pd-standard'
+        gce_root_disk_size_monitor: 50
+        gce_n_local_ssd_disk_monitor: 0
+        scylla_repo: 'REPO_FILE_PATH'
+        us_east_1:
+          gce_datacenter: 'us-east1-b'
+    aws: !mux
+        cluster_backend: 'aws'
+        user_credentials_path: '~/.ssh/scylla-qa-ec2'
+        instance_type_loader: 'c4.2xlarge'
```
Comment (roydahan) [inline tests/longevity-2mv-backpressure-4d.yaml]: please change to equivalent c5 (need to check the spec of it)

File: tests/longevity-2mv-backpressure-4d.yaml [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+experimental: 'true'
+#append_conf: 'enable_sstable_data_integrity_check: true'
+append_scylla_args: '--blocked-reactor-notify-ms 500'
+
+# aws instance_provision: on_demand|spot_fleet|spot_low_price|spot_duration
+instance_provision: 'spot_low_price'
+
+# scylla-manager configuration
+# if running on aws and use_mgmt is true, the monitor image should not contain scylla
+use_mgmt: false
```
Comment (roydahan) [inline tests/longevity-2mv-backpressure-4d.yaml]: Change to true and the details below according to the other updated Yamls.

File: tests/longevity-2mv-backpressure-4d.yaml [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+            "cassandra-stress user profile=/tmp/c-s_profile_2mv_2queries.yaml ops'(mv_p_read1=1,mv_p_read2=1)' cl=QUORUM duration=5760m -port jmx=6868 -mode cql3 native -pop seq=1..250000000 -rate threads=10"]
+n_db_nodes: 4
+n_loaders: 4
+n_monitor_nodes: 1
+user_prefix: 'longevity-2mv-backpressure-4d-VERSION'
+nemesis_class_name: 'ChaosMonkey'
+nemesis_interval: 30
+nemesis_during_prepare: 'false'
+failure_post_behavior: keep
+space_node_threshold: 644245094
```
Comment (roydahan) [inline tests/longevity-2mv-backpressure-4d.yaml]: Is it needed? will it start nemesis?

File: tests/longevity-2mv-backpressure-4d.yaml [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+ip_ssh_connections: 'private'
+#When the load is too heavy for one lader when using MULTI-KEYSPACES, the load is spreaded evenly across the loaders (round_robin)
+round_robin: 'false'
+experimental: 'true'
+#append_conf: 'enable_sstable_data_integrity_check: true'
```
Comment (roydahan) [inline tests/longevity-2mv-backpressure-4d.yaml]: please remove it

File: tests/longevity-2mv-backpressure-4d.yaml [hunk 5]
```diff
@@ -0,0 +1,77 @@
+test_duration: 5760
```
Comment (roydahan) [inline tests/longevity-2mv-backpressure-4d.yaml]: Make it a bit bigger than 5760 which is the duration of the stress.
It should be test duration = setup time + prepare write time + stress time + short buffer.

Comment (juliayakovlev) [reply tests/longevity-2mv-backpressure-4d.yaml]: Changed to 4.5 days (6480 min)

File: tests/longevity-2mv-backpressure-4d.yaml (hunk 3 above)
Comment (juliayakovlev) [reply tests/longevity-2mv-backpressure-4d.yaml]: Yes, after the https://github.com/scylladb/scylla-cluster-tests/pull/769 will be merged

**Comment by juliayakovlev** [comment]
@roydahan  Please, review

**Pull request merged**